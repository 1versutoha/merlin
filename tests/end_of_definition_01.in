["tell","source","let f =\n"]
["tell","more","func\n"]
["tell","more","arg\n"]
["tell","more","let t =\n"]
