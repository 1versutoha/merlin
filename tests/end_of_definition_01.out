["return",null]
["return",null]
["return",null]
["return",{"line":3,"col":3}]
